### ISSUE TITLE
bot.startConvertsation is broken (Facebook)

### ISSUE THREAD

**Issue by vnxdev** [howdyai/botkit#1056]
I've got some code that was working fine with 0.5.8 version. I've pulled the latest version from this repo and the bot.startConvertsation is stuck at creating conversation:

```
info: [Start]  0  Task for  2034620566571234 in 2034620566571234
info: >   [Start]  0  Conversation with  2034620566571234 in 2034620566571234
```

The sample code:
```
bot.startConversation(message, (err, convo) => {
    convo.ask({
          text: 'ASK'
        }, (response, convo) => {
          console.log(response)
          convo.next()
        })
      })
```

Bot doesn't send the message with question and stops receiving any events from Facebook. 
This code works as expected with 0.5.8.

**Comment by benbrown** [comment]
@vnxdev This has been resolved in v0.6.2. 

Thank you for your bug report.  This was due to a missing call to `startTicking()`

**Comment by benbrown** [comment]
Please let us know if updating to 0.6.2 DOESN'T resolve your issue
